config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
We should be using community images and not building our own. Is there a community image for rust that we could use?